### ISSUE TITLE
GLIMPSE2_ligate_static Failed to open vcf

### ISSUE THREAD

**Issue by zxy1555847** [odelaneau/GLIMPSE#147]
Hi!
I fallowed this [tutorial](https://odelaneau.github.io/GLIMPSE/docs/tutorials/getting_started/) to impute my low-coverage data(vcf) using 1000GP reference panel.
The first few steps went well, until STEP4 Ligate the imputed chunks to get a chromosome-wide file .
The vcf file of every chunk  is generated from GLIMPSE2_phase. 
However, if I turn vcf to bcf , GLIMPSE2_ligate will run normally. So I think it maybe a interesting little bug.

**Comment by srubinacci** [comment]
Hi, 
Thank you for reporting this. I believe it's a wanted feature: while GLIMPSE accepts VCF file format as an output, we tend to avoid it, and use compressed VCF files (vcf.gz) or compressed BCF files, as they can be indexed for random-access. The ligation program makes use of the csi/tbi indeces and it throws an error.
I reckon a more user-friendly message should be displayed, or maybe we should try to make the ligate program to work on VCF files directly.

**Comment by zxy1555847** [comment]
Yes, I think vcf.gz or bcf is better than vcf. It's just that this is the first time I've used GLIMPSE and this error of not being able to open vcf file has really distressed me all morning, because it says in the --help that vcf is acceptable, and it's the vcf generated in the previous step of GLIMPSE. 
I think modifying the error message is the best solution, vcf file is not necessary, just prompt the user that vcf.gz is needed.
